msg/RoverAckermannSetpoint.msg

```diff
@@ -0,0 +1,6 @@
+uint64 timestamp # time since system start (microseconds)
+
+float32 speed # [-1, 1] Normalized speed setpoint in body forward direction
+float32 steering # [-1, 1] Normalized steering angle setpoint (positive: right, negative: left)
```

Comment:
The main reason `RoverAckermannControl` exists is because I tried to keep the module in line with the differential module. But I would actually say it makes more sense to merge it into `RoverAckermann` since that would eliminate the need for the `RoverAckermannSetpoint` entirely and things like yaw rate control (and slew rates for steering/throttle) could be functions in `RoverAckermann`. Maybe @PerFrivik has some thoughts on this?

Reply:
Discussed it with Per and changed it in https://github.com/PX4/PX4-Autopilot/pull/23024/commits/bedca441065af74fa05d416a3dc603aaf4e98d22.